<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendDataDialog</class>
 <widget class="QDialog" name="SendDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据发送</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="title">
      <string>帧发送</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label_21">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="text">
           <string>帧ID:0x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QtMaterialTextField" name="editId">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>22</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="text">
           <string>发送次数:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinSendCount">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <widget class="QtMaterialCheckBox" name="chkInc">
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="text">
         <string>ID递增</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="text">
           <string>帧格式:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboFormat">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="text">
           <string>数据长度:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboLength">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>帧类型:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboFrameType">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboProtocolType">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="text">
           <string>名称(可选):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QtMaterialTextField" name="editName">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>22</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="4">
       <widget class="QtMaterialRaisedButton" name="btnSend">
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="text">
         <string>立即发送</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="text">
           <string>每次发送帧数:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinSendCountOnce">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="text">
           <string>数据:0x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QtMaterialTextField" name="editData">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="4">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="text">
           <string>每次间隔(ms):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinInterval">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="4">
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="text">
           <string>发送方式:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboSendType">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="title">
      <string>列表发送</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTableView" name="tableView">
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QtMaterialRaisedButton" name="btnSendList">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="text">
           <string>列表发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialRaisedButton</class>
   <extends>QPushButton</extends>
   <header>qtmaterialraisedbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialCheckBox</class>
   <extends>QCheckBox</extends>
   <header>qtmaterialcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialTextField</class>
   <extends>QLineEdit</extends>
   <header>qtmaterialtextfield.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
